src/context_store/retrieval/post_processor.py

```
@@ -0,0 +1,156 @@
+
+            # トークン推定
+            if ascii_ratio >= 0.9:
+                # 英語主体
+                estimated_tokens = len(content) // 4
+                safety_margin = 1.5
+            else:
+                # 日本語等マルチバイト
+                estimated_tokens = len(content) // 3
+                safety_margin = 3.0
+
+            # 安全側（過大推定）
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)